test/usersignup.go

```diff
@@ -7,11 +7,11 @@ import (
 
 	"github.com/codeready-toolchain/toolchain-common/pkg/condition"
 	"github.com/codeready-toolchain/toolchain-common/pkg/states"
+	"github.com/gofrs/uuid"
 
 	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
 	"github.com/codeready-toolchain/toolchain-common/pkg/test"
 
-	uuid "github.com/satori/go.uuid"
```

Comment:
it would be much better to do all these changes in a separate PR

Reply:
We had previously decided to update this dependency any time we came across it, there is no impact on the main focus of this PR so no harm in including the update here.